guides/model-state.md

```diff
@@ -0,0 +1,13 @@
+## Avoid Local/UI state on Models
+
+Models live on the `store` service, and all services hold "application state."
+UI or local state should not be entangled with application state by adding
+it to a model.
```

Comment:
Also it's worth noting that you were able to explain your sentiment clearly, without talking about how the store service really owns the identity map. This is one of several places in this document where "extra" information muddles what could otherwise be a clear concept.